rma_sale/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <block name="catalog_setting_container" position="inside">
+                <div
+                    class="col-12 col-lg-6 o_setting_box"
+                    title="Show portal RMA request in a single page"
+                >
+                    <div class="o_setting_left_pane">
+                        <field name="show_full_page_sale_rma" />
+                    </div>
+                    <div class="o_setting_right_pane">
+                        <label
+                            for="show_full_page_sale_rma"
+                            string="Single page RMA request"
```

Comment:
I struggled to find this feature that I know existed :sweat_smile: 

Can you:

- Put it in the RMA config section
- Add a configuration README entry